File: app/app/src/main/app/src/main/java/com/realmofaruna/MainActivity.java
package com.realmofaruna;

import android.app.Activity;
import android.os.Bundle;
import android.webkit.WebSettings;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        WebView webView = new WebView(this);

        WebSettings settings = webView.getSettings();
        settings.setJavaScriptEnabled(true);
        settings.setDomStorageEnabled(true);
        settings.setAllowFileAccess(true);
        settings.setAllowContentAccess(true);

        webView.setWebViewClient(new WebViewClient());

        webView.loadData(
            "<html><body style='background:black;color:red;text-align:center;padding-top:100px'><h1>WEBVIEW BARU</h1></body></html>",
            "text/html",
            "UTF-8"
        );

        setContentView(webView);
    }
    }

File: app/app/src/main/app/src/main/java/com/realmofaruna/MainActivity.kt
package com.realmofaruna

import android.app.Activity
import android.os.Bundle
import android.webkit.WebView

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val webView = WebView(this)

        webView.settings.javaScriptEnabled = true
        webView.settings.domStorageEnabled = true

        webView.loadDataWithBaseURL(
            null,
            """
            <!DOCTYPE html>
            <html>
            <body style="background:#10131a;color:white;text-align:center;padding-top:100px">
                <h1>Realm Of Aruna</h1>
                <p>WebView berhasil berjalan!</p>
            </body>
            </html>
            """.trimIndent(),
            "text/html",
            "UTF-8",
            null
        )

        setContentView(webView)
    }
}
